README.md

```diff
@@ -1,70 +1,65 @@
-* Integration with model serving frameworks like [NVIDIA’s Triton Inference Server](https://github.com/NVIDIA/tensorrt-inference-server) and [Tensorflow Serving](https://github.com/tensorflow/serving) to make model deployment easy. (WIP)
-* Statistical monitoring of the dataset for distributional shift and outlier detection during production training or inference. (WIP)
+The library is designed to be interoperable with both PyTorch and TensorFlow using dataloaders that we have developed as extensions of native framework code. NVTabular provides the option to shuffle data during preprocessing, allowing the dataloader to load large contiguous chunks from files rather than individual elements. This allows us to do per epoch shuffles orders of magnitude faster than a full shuffle of the dataset. Loading tabular data is almost always a training bottleneck. In our benchmarking, we've seen 10x improvements in training time on the GPU relative to the native dataloaders.
 
-The library is designed to be interoperable with both PyTorch and TensorFlow using dataloaders that we have developed as extensions of native framework code. NVTabular provides the option to shuffle data during preprocessing, allowing the dataloader to load large contiguous chunks from files rather than individual elements. This allows us to do per epoch shuffles orders of magnitude faster than a full shuffle of the dataset.  Loading tabular data is almost always a training bottleneck and in our benchmarking we have seen 10x improvements in training time on GPU relative to the native dataloaders. 
+Extending beyond model training, we plan on integrating with model-serving frameworks like NVIDIA’s Triton Inference Server. This will only create a clear path to production inference for these models and allow the feature engineering and preprocessing steps performed on the data during training to be automatically applied to incoming data during inference.
 
-Extending beyond model training, we plan to provide integration with model serving frameworks like NVIDIA’s Triton Inference Server, creating a clear path to production inference for these models and allowing the feature engineering and preprocessing steps performed on the data during training to be automatically applied to incoming data during inference.
-
-Our goal is faster iteration on massive tabular datasets, both for experimentation during training, and also for production model responsiveness. 
+Our goal is faster iteration on massive tabular datasets, both for experimentation during training, and also for production model responsiveness.
 
 ### Getting Started
 NVTabular is available in the NVIDIA container repository at the following location: http://ngc.nvidia.com/catalog/containers/nvidia:nvtabular.
 
-Currently we have the beta release (0.2) container, you can pull the container using the following command:
+The beta (0.3) container is currently available. You can pull the container by running the following command:
```

Comment:
This container isn't available yet - should we hold off on merging this until monday when we ship the container?

Reply:
Sounds good.